project/scripts/release

```diff
@@ -299,6 +299,7 @@ else
   RELEASE_OPT="-Dakka.genjavadoc.enabled=true"
 fi
 try sbt $RELEASE_OPT +buildRelease
+try sbt -Dakka.build.scalaVersion=2.11.12 -D$RELEASE_OPT buildRelease
```

Comment:
one drawback here is that the pgp passphrase must be entered again, but I guess we can live with that (this will be removed for 2.6)

Reply:
I think that is ok for now